dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_memory.cc

```diff
@@ -12,18 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "nvjpeg_memory.h"
+#include "dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_memory.h"
```

Comment:
Why is this header on top?

Reply:
That's the order that the linter recommends. First goes "your"  header (the one named the same as the current file), then sytem C headers, the system C++ headers, then the rest.